Libplanet/Blockchain/BlockChain.PoS.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
+using Libplanet.PoS;
+using Libplanet.PoS.Model;
+
+namespace Libplanet.Blockchain
+{
+    public partial class BlockChain<T>
+    {
+        public ValidatorSet GetBondedValidators(BlockHash? hash = null)
+        {
+            IValue value = GetState(ValidatorSet.BondedAddress, hash);
+            return new ValidatorSet(value);
+        }
+
+        public Validator Proposer(long height, int round)
+        {
+            IValue value = GetState(ReservedAddress.BondedValidatorSet);
+            ValidatorSet bondedValidatorSet = new ValidatorSet(value);
+            ValidatorPower bondedValidator =
+                bondedValidatorSet[(int)((int)(height + round) % bondedValidatorSet.Count)];
+            return new Validator(
+                bondedValidator.OperatorPublicKey.ToAddress(),
+                bondedValidator.OperatorPublicKey);
+        }
+
+        public List<PublicKey> BondedValidatorPubKey(BlockHash? hash = null)
+        {
+            ValidatorSet validatorSet = GetBondedValidators(hash);
+            return validatorSet.Set.Select(x => x.OperatorPublicKey).ToList();
+        }
+    }
+}
```

Comment:
I'm thinking about right place to generate proposer.
It can be done on various places, but seems to handle this on block action(same procedure as validator set election, right after it) would be suitable, since selecting validator set and proposer at the same time make sense,
and since VRF proof hash can be used in various ways, it would be better to inject VRF proof to `IActionContext`.
We can talk about it later, though. :)